service-base/src/main/java/io/narayana/lra/LRAConstants.java

```diff
@@ -65,6 +65,7 @@ public final class LRAConstants {
      */
     public static final long PARTICIPANT_TIMEOUT = 2;
     public static final String ALLOW_PARTICIPANT_DATA = "lra.participant.data";
+    public static final String WAIT_CALLBACK_TIMEOUT = "lra.participant.callback.timeout";
```

Comment:
I think this shouldn't mention participant and rather it should be clear that it's only intended for the TCK.

Reply:
I'm also not sure if it would make sense to move this to NarayanaLRARecovery since it's only used there.

Reply:
I would prefer it to move, this class is in the public API. Is that possible @marcosgopen ?

Reply:
SUre, I'll move it to NarayanaLRARecovery. I was not sure that was needed for tck tests only.